modules/tide_edit_protection/tide_edit_protection.module

```diff
@@ -142,3 +143,38 @@ function tide_edit_protection_preprocess_status_messages(&$variables) {
     }
   }
 }
+
+/**
+ * Implements hook_entity_predelete().
+ */
+function tide_edit_protection_entity_predelete(Drupal\Core\Entity\EntityInterface $entity) {
+  if (PHP_SAPI == 'cli') {
```

Comment:
the predelete hook of content_lock is totally wrong and it causes more issue upon entity deletion not just via CLI. Hence we should release the lock regardless of the situation.
```php
$lock_service->release($entity_id, LanguageInterface::LANGCODE_NOT_SPECIFIED);
```
Just this line is enough. No need to check for CLI, no need to fetch the lock.